playbooks/compute/nto/roles/configurecluster/tasks/create_directory.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Check if folder exists
+  ansible.builtin.stat:
+    path: "{{ dir_path }}"
+  register: register_name
```

Comment:
I don't see a change here in the name, I think the name doesn't match the task intention.

Reply:
changed